### ISSUE TITLE
Zeal latest version not working on Windows 10 version 10.0.16299.192

### ISSUE THREAD

**Issue by abel406** [zealdocs/zeal#907]
Hi.I installed zeal using the latest installer zeal-0.5.0-windows-x64.msi and for some reason it does not work. Aparently it is related with the browser engine. The messsage is "A simple documentation browser. dejó de funcionar" In plain english "A simple documentation browser. stopped working".

Please, some help. Thanks.

**Comment by abel406** [comment]
I tried to debug it and it shows an error with Qt5Core.dll. Could this have something to do with the reason for malfunctioning ?

**Comment by tserafim** [comment]
Have you checked #889?

**Comment by trollixx** [comment]
Duplicate of #889.
